pkg/apis/submariner.io/v1/types.go

```diff
@@ -95,6 +95,7 @@ const (
 	PreferredServerConfig   = "preferred-server"
 	PublicIP                = "public-ip"
 	UsingLoadBalancer       = "using-loadbalancer"
+	ForceTCPMssClamping     = "submariner.io/tcp-mss-clamping"
```

Comment:
```suggestion
	ForceTCPMssClamping     = "submariner.io/tcp-clamp-mss"
```

On a second thought, "tcp-mss-clamping" seems to convey a boolean value like true/false. Shall we use "tcp-clamp-mss" as it can convey an assignment.

Reply:
I'm not good at choosing names :-( , let's use 'tcp-clamp-mss' as u proposed